app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -7,12 +8,12 @@ class PagesController < Alchemy::Admin::BaseController
         except: [:show]
 
       before_action :load_page,
-        only: [:show, :info, :unlock, :visit, :publish, :configure, :edit, :update, :destroy, :fold]
+        only: [:show, :info, :unlock, :visit, :publish, :configure, :edit, :update, :destroy, :fold, :tree]
```

Comment:
Metrics/LineLength: Line is too long. [107/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)